### ISSUE TITLE
Fails to parse certain tags

### ISSUE THREAD

**Issue by jeroen** [yihui/Rd2roxygen#23]
Trying to parse:

```
\title{Convert \R{} objects to/from JSON}
```

turns into 

``` r
$title
[1] "Convert  objects to/from JSON"
```

**Comment by yihui** [comment]
See #22, i.e. if you can fix it, great and pull requests welcome; otherwise, I have no intention to make this package perfect.

**Comment by joshuaulrich** [comment]
This is fixed by #35.
